### DISCUSSION TITLE
Use factory edges?

### DISCUSSION THREAD

**Discussion by QueueAK** [lairdubois/lairdubois-opencutlist-sketchup-extension#323] [Ideas]
I would like an option to use the factory edges or edit the edge off cuts.

I'm doing some rough shelving with4x8 plywood. I noticed if I can't set the shelves to 1' 11 7/8" and get two pieces out of one sheet. The off cut is 1' 11" and there seems to be a boarder around the sheet which is cut off. When making nice cabinetry I do not use the factory edges, but with rough work it is just fine.

**Comment by mobilarte** [comment]
This option already exists. In the Cutting Diagram -> Configuration, just below the Blade Thickness, you may define the Trimming Size.

![Clipboard01](https://user-images.githubusercontent.com/24858423/128300842-0d45d4d1-a953-49e9-a49f-383292b1e66d.jpg)

**Reply by QueueAK** [reply]
Wow, I completely missed that page of settings. Thanks.
